llumnix/manager.py

```diff
@@ -372,7 +344,7 @@ def scale_up(self,
                 self.pending_rebuild_migration_instances += 1
 
         if indeed_update:
-            self.global_scheduler.scale_up(instance_ids, instance_types)
+            await self.global_scheduler.scale_up(instance_ids, instance_actor_handles, instance_types)
```

Comment:
there is potential bug here, you add the instance to self.instances, but global scheduler has not been updated.